content/rc/api/_index.md

```diff
@@ -1,43 +1,38 @@
 ---
-Title: API
-description:
+Title: REST API
+description: Describes the Redis Cloud REST API and links to additional info.
```

Comment:
I wouldn't end this with a period, since it's a sentence fragment. Just my two cents

Reply:
Fixed.